src/components/victory-errorbar/errorbar.js

```diff
@@ -27,99 +27,43 @@ export default class ErrorBar extends React.Component {
     borderWidth: PropTypes.number
   };
 
-  renderBorder() {
+  render() {
     const {
       x,
       y,
       errorX,
       errorY,
-      borderWidth
+      scale
     } = this.props;
 
-    return (
-      <g>
-        <line
-          {...this.props.events}
-          style={this.props.style}
-          x1={x + (isArray(errorX) ? errorX[1] : errorX)}
-          x2={x + (isArray(errorX) ? errorX[1] : errorX)}
-          y1={y - borderWidth}
-          y2={y + borderWidth}
-        />
-        <line
-          {...this.props.events}
-          style={this.props.style}
-          x1={x - (isArray(errorX) ? errorX[1] : errorX)}
-          x2={x - (isArray(errorX) ? errorX[1] : errorX)}
-          y1={y - borderWidth}
-          y2={y + borderWidth}
-        />
-        <line
-          {...this.props.events}
-          style={this.props.style}
-          x1={x - borderWidth}
-          x2={x + borderWidth}
-          y1={y - (isArray(errorY) ? errorY[1] : errorY)}
-          y2={y - (isArray(errorY) ? errorY[1] : errorY)}
-        />
-        <line
-          {...this.props.events}
-          style={this.props.style}
-          x1={x - borderWidth}
-          x2={x + borderWidth}
-          y1={y + (isArray(errorY) ? errorY[1] : errorY)}
-          y2={y + (isArray(errorY) ? errorY[1] : errorY)}
-        />
-      </g>
-    );
-  }
-
-  render() {
-    const {
-      x,
-      y,
-      errorX,
-      errorY
-    } = this.props;
+    const rangeX = scale.x.range();
+    const rangeY = scale.y.range();
+    const positiveErrorX = x + (isArray(errorX) ? errorX[0] : errorX);
+    const negativeErrorX = x - (isArray(errorX) ? errorX[1] : errorX);
+    const positiveErrorY = y + (isArray(errorY) ? errorY[1] : errorY);
+    const negativeErrorY = y - (isArray(errorY) ? errorY[0] : errorY);
+    const errorTop = positiveErrorY >= rangeY[0] ? rangeY[0] : positiveErrorY;
```

Comment:
@boygirl instead of building a standalone `getDomain` function in Error-bar, which you mention in the PR. I've add a if else statement to check if it is out. If it is I'll set to the minimum or maximum range to it.